services/agent-opt-worker/main.py

```
@@ -0,0 +1,527 @@
+            logger.warning(f"[{template}] empty eval_results")
+            return {"error": f"Empty result from SDK for {template}"}
+        er = result.eval_results[0]
+        er_dict = {k: v for k, v in vars(er).items() if not k.startswith("_")} if hasattr(er, "__dict__") else str(er)
+        logger.info(f"[{template}] raw result: {er_dict}")
+
+        # SDK returns: output="Passed"/"Failed", output_type="Pass/Fail",
+        # reason=explanation, runtime=ms, eval_id=uuid
+        output = getattr(er, "output", "")
+        reason = getattr(er, "reason", "") or ""
+        score = getattr(er, "score", None)
+        runtime_ms = getattr(er, "runtime", None)
```

Ruff (0.15.1)

[error] Local variable `runtime_ms` is assigned to but never used

Remove assignment to unused variable `runtime_ms`

(F841)